webapp/src/main/java/com/box/l10n/mojito/service/security/user/UserService.java

```diff
@@ -40,6 +51,15 @@ public class UserService {
 
   @Autowired AuditorAwareImpl auditorAwareImpl;
 
+  @Autowired(required = false)
+  ServiceDisambiguator serviceDisambiguator;
+
+  @Autowired HeaderSecurityConfig headerSecurityConfig;
+
+  @Autowired PagerDutyIntegrationService pagerDutyIntegrationService;
```

Comment:
I think `PagerDutyIntegrationService` shouldn't be a required dependency here as some deployments may not have it configured, meaning Spring will fail to start up.

Same comment for `HeaderSecurityConfig`, don't think it should be required dependency. You'll need to update the `getServiceAccountUser` code to handle these injections being null

Reply:
That's fair. I've updated this.